src/Meshes/mesh.ts

```diff
@@ -2949,18 +2962,6 @@ export class Mesh extends AbstractMesh implements IGetSetVerticesData {
      * @returns a new InstancedMesh
      */
     public createInstance(name: string): InstancedMesh {

```

Comment:
This is the important goal we are after. No per-source mesh operation mutates the Geometry so we don't need to make it unique in order to create instances. This lets us clone a template-mesh many times. From each clone we can create instances. They all share the same heavy Geometry obejct.

The reason we need a template is that cloning doesn't work well once we start having instances.

Reply:
I genuinely love it!!!!!